versions.gradle

```diff
@@ -9,7 +9,7 @@ ext {
     versionCode = 63
     versionName = '4.1.2'
     minSdkVersion = 21
+    compileSdkVersion = "android-S"
```

Comment:
Do we need this change?

Reply:
I think so, yes. It's just a different notation for version after 30.